use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -323,17 +372,28 @@ main(int argc, char *argv[])
             resp.counter = ++clientp->client_ctr;   // Do Increment
             clientp->num_ops++;                     // Track # of operations
 
-#if L3_ENABLED
+            // Record time-consumed. (NOTE: See clarification below.)
+#  if defined(L3_LOGT_SPDLOG)
+            spd_logger->info("Server spdlog: "
+                             "Increment: ClientID={}, Counter{}.",
+                             resp.clientId, resp.counter);
+
+#  elif defined(L3_LOGT_SPDLOG_BACKTRACE)
+
+            spdlog::debug("Server spdlog-Backtrace: "
+                         "Increment: ClientID={}, Counter={}.",
+                         resp.clientId, resp.counter);
+
+#elif L3_ENABLED
```

Comment:
I note an inconsistently here -- for speedlog the macro need only be defined, but for L3 etc it must be defined to non-zero. (Not a big deal.)

Reply:
I understand ... here are my reasons.

Actually, in Makefile all these #if'defs are issued with `-D<flag> ` clause, so, technically, I could use `#if defined(L3_ENABLED)` also.

For `L3_ENABLED`, we have some invocations like:

```
CC=gcc LD=g++ L3_ENABLED=0 make client-server-perf-test  # Baseline

L3_ENABLED=0 L3_LOGT_SPDLOG=1 make client-server-perf-test  # C++ spdlog logging
```

Where one has to explicitly turn OFF `L3_ENABLED` by setting it to 0.

So, there the use of `#if L3_ENABLED` would not be fully indicative of the fact that sometimes this env-var setting at `Makefile`-context is acting like a boolean.

Env-var setting `L3_LOGT_SPDLOG=1,2` translates to one of `-DL3_LOGT_SPDLOG` or `-DL3_LOGT_SPDLOG_BACKTRACE` . Here, `#if defined()` seemed appropriate.

I am not an expert in figuring out these conventions & rules for when to use which form.

I'd like to get this feature-support in and we can revisit holistically the right code-patterns, starting from env-vars thru Makefile `-D<flags>` and fix things as needed.